share/get_test.go

```diff
@@ -237,7 +237,7 @@ func TestCollectLeavesByNamespace_AbsentNamespaceId(t *testing.T) {
 	t.Cleanup(cancel)
 	bServ := mdutils.Bserv()
 
-	shares := RandShares(t, 16)
+	shares := RandShares(t, 1024)
```

Comment:
can we just extract into var

Reply:
ubernit: but might be a const at the top of the test, but still inside

Reply:
It is single value and already in top of the test. Why would you want var for to store it?